static/js/stream_edit.js

```diff
@@ -737,8 +737,24 @@ export function initialize() {
             is_web_public: stream.is_web_public,
         });
         const stream_name = stream_data.maybe_get_stream_name(stream_id);
+
+        const is_new_stream_notification_stream =
+            stream_id === page_params.realm_notifications_stream_id;
+        const is_signup_notification_stream =
+            stream_id === page_params.realm_signup_notifications_stream_id;
+        const is_report_notification_stream =
+            stream_id === page_params.realm_report_message_stream_id;
```

Comment:
This is considering the changes done at [#21821](https://github.com/zulip/zulip/pull/21821). But if decided to merge this PR before that one, we should not access `realm_report_message_stream_id` because it doesn't exist yet although it's working because the comparison is false even if `realm_report_message_stream_id` is undefined.